map/src/components/Map/Map.js

```diff
@@ -886,30 +889,30 @@ const Map = ({data}) => {
             });
 
         });
-    
-    }, [])
+
+    }, [regionsIndex, repIndex])
 
     return (
         // container for the entire app
-        <div class="wrapper">
+        <div className="wrapper">
```

Comment:
replaced `class` => `className` as per the React recommendation: https://reactjs.org/docs/dom-elements.html#classname